__DEFINES/mobs.dm

```diff
@@ -22,3 +22,6 @@
 #define MESSENGER_BAG 5
 
 #define SHOW_HELD_ITEM_AND_POINTING_DELAY 0.7 SECONDS
+
+#define YES_THERE_IS_A_ROCKERNAUT 1
+#define ITS_A_BIG_ROCKERNAUT_WOW  2
```

Comment:
Don't let Damian find out about these defines names he is pretty picky about them